.github/workflows/test_client.yaml

```diff
@@ -49,4 +50,9 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
 
     - name: Check types with mypy
-      run: mypy bastionai
+      run: |
+        rm -rf bastionai/pb/remote_torch*
```

Comment:
Maybe use path exceptions in mypy.ini?

Reply:
I tried but I had issues with that.
I will retry and see if it works